### ISSUE TITLE
Is this still working though?

### ISSUE THREAD

**Issue by savokiss** [Sh4yy/cloudflare-email#6]
I have deploy my this worker into production.

And added a `SPF` record and a `_mailchannels` record to my domainA.

And added a custom domainB to my worker.

Then try  to POST `https://domainB/api/email` the body below:

```json
{
	"to": "myown@gmail.com",
	"from": "support@domainA",
	"subject": "Hello World",
	"text": "Hello World"
}
```

But it always showed `Internal Server Error`.

Is there anything I missing to use this email sender?

**Comment by Sh4yy** [comment]
Have you set up the new _mailchannels DNS record?

https://github.com/Sh4yy/cloudflare-email#setup-spf

**Comment by savokiss** [comment]
Yes, I have set up this record too. But it is still not working, just respond the `Internal Server Error` message.

**Comment by andreresende** [comment]
Same error here. Error 500

**Comment by V-ed** [comment]
It happened to me too. I initially used the `Deploy with Workers` button and that never worked. Upon inspecting the code on cloudflare quick edits feature, I noticed the worker deployed was actually the default hello world. Maybe I configured it badly, dunno.

I then tried to use the 4 steps of cloning the repo locally to deploy it with wrangler and after properly setting up the SPF record as well as the now required TXT [Domain Lockdown error](https://support.mailchannels.com/hc/en-us/articles/16918954360845-Secure-your-domain-name-against-spoofing-with-Domain-Lockdown) (and since I was already using cloudflare to redirect my domain's email, I had to merge them so it looked at the end like this : `v=spf1 a mx include:relay.mailchannels.net include:_spf.mx.cloudflare.net ~all`), and after setting up both it seems to work fine for me!

Make sure to also always use a `from` mail coming from your domain, for some reason I thought I could send emails with the `from` field as any email, obviously that doesn't work haha

---

So yeah, maybe the `Deploy with Workers` button is broken, that I can't confirm, but it works for me after passing by and upvoting this issue initially.

**Comment by andreresende** [comment]
Yes, it working here after configuring domain lockdown. I think it is mandatory now.

**Comment by V-ed** [comment]
Confirmed mandatory for Cloudflare workers (from [Domain Lockdown error](https://support.mailchannels.com/hc/en-us/articles/16918954360845-Secure-your-domain-name-against-spoofing-with-Domain-Lockdown)) :

![image](https://github.com/Sh4yy/cloudflare-email/assets/16652915/dff240db-717d-4d8d-afd6-0576a0208e0f)
